src/main/java/reborncore/api/items/InventoryUtils.java

```diff
@@ -58,29 +67,46 @@ public static ItemStack insertItemStacked(Inventory inventory, ItemStack input,
 		return stack;
 	}
 
-	public static ItemStack insertItem(ItemStack input, BlockEntity blockEntity, Direction direction){
+	public static ItemStack insertItem(ItemStack input, Inventory inventory, Direction direction) {
```

Comment:
What’s the reasoning behind changing to an inv here? Might not be a bad idea to add the BE version back? I have a feeling the BE version is used.

Reply:
My reasons are:
 - the method itself expects Inventory
 - after rewrite i need a method that accepts Inventory directly 
 - all reborn machines implemented Inventory
 - after 2 hours if play testing it works just fine
**But** i’ll double check all usages soon an then let you know here

Reply:
Just checked. This function only used in SlotConfig (here), which is nice actually.